### ISSUE TITLE
формы от русских заимствований

### ISSUE THREAD

**Issue by veramal** [tuffnatty/suddenly#225]
От лексемы "механик", которая есть в словаре, не анализируется посессив - ни форма _механигі_, ни _механиғы_ (правильная - первая).
От лексемы "машинист" анализируется _машинизы_, но правильная форма - _машинизі_. В русских заимствованиях гармония по переднему ряду, если гласная последнего слога - и.

**Comment by tuffnatty** [comment]
Кажется, в документе про это нигде не написано.

**Comment by veramal** [comment]
Есть на стр. 9, п. 3.1. Там есть еще про сагайские формы, но их мы нормализуем все равно.

**Comment by tuffnatty** [comment]
Вроде удается исправить, но тогда ломается тестовая форма "тариф+ы", так и надо?

**Comment by veramal** [comment]
Да, так и надо. На сайте хакасской газеты удалось найти форму _тарифтер_, т.е. _тариф_ переходит в передний ряд, как полагается. А _тариф+ы_ вообще должно давать _таривi_, но точную форму найти ни в каком варианте не удалось. В корпусе из похожих случаев вроде бы только _калиф_ из Ходжи Насреддина ведет себя непоследовательно (_калифтiң_, но _калифы_) - спишем на старые нормы орфографии и забьем.

**Comment by veramal** [comment]
Правильные формы с передней гласной анализируются, неправильные с задней сломаны.

**Comment by veramal** [comment]
Немного другая проблема в этом же направлении: _механиктер_ работает, _кооперативтер_ нет. _Бефстрогановтар_ тоже нет, а _заводтар_ и _антропологтар_ да. Видимо, конечное _в_ не включено в оглушение.

**Comment by veramal** [comment]
Анализируются все указанные (как верные) словоформы: _механигі, машинизі, механиктер, кооперативтер, бефстрогановтар, заводтар, антропологтар_.
